wls-auth-service/src/main/resources/db/migrations/oracle/V10_0_addAdminPermissionsGetKonfigurierteWahltage.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO Permission
+VALUES ('00000000-0000-0009-0000-000000000001', 'Admin_BUSINESSACTION_GetKonfigurierteWahltage');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0009-0000-000000000001');
```

Comment:
```suggestion
VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0010-0000-000000000001');
```
siehe oben, permission id bereits im V9 file vergeben